### DISCUSSION TITLE
Should we rename the reminders domain object?

### DISCUSSION THREAD

**Discussion by Pablito2020** [ToDoTurtle/ToDoTurtle#11] [Announcements]
For now, we are calling a reminder an object that has title and description, but I think that a better way to name that would be a Note.

**A reminder is a note with a notification deadline.**
**A Todo is just a note with a deadline**

So, our application could have:
- Notes => Which are just plain text.
- Reminders => Which are notes that have notifications related to them
- Todos => Which are notes that have a deadline assigned to them.

What do you think? We should refactor the package and classes names before it is a pain in the ass?

**Comment by oriolagobat** [comment]
I agree with the change, and I think that refactoring now is the best thing we can do.

**Reply by oriolagobat** [reply]
As soon as the current PR is accepted, I'll issue a new one with the change.

**Comment by FerranAD** [comment]
I agree, from now on I only speak Notes 🤑.
